src/ring_theory/laurent_series.lean

```diff
@@ -99,24 +99,36 @@ begin
     coe_power_series],
 end
 
+@[simp] lemma of_power_series_C (r : R) :
+  of_power_series ℤ R (power_series.C R r) = hahn_series.C r :=
```

Comment:
Is this true more generally for `hahn_series.of_power_series`? If so, can you promote `hahn_series.of_power_series` to an `alg_equiv`?

Reply:
This looks like it should be straightforward, as the alg_hom versions of all the components of `hahn_series.of_power_series` seem to exist.

Reply:
Ah nevermind, I forget that `power_series.C` is only `algebra_map` when `R` is commutative.